hal/common/file.hpp

```diff
@@ -142,14 +142,21 @@ class File
 	 *
 	 * \return 	success	(always false)
 	 */
-	 bool flush();
+	bool flush();
 
 	/**
 	 * \brief 	write newline character to file ('\n')
 	 *
 	 * \return 	success
 	 */
-	 bool newline();
+	bool newline();
+
+	/**
+	 * \brief	sync the file
```

Comment:
Why? According to what you say, `sync` writes temporary memory to the file. And the documentation of flush in File and Serial say:
```
	/**
	 * \brief 	flush buffer to file; Not implemented;
	 *
	 * \return 	success	(always false)
	 */
	 bool File::flush();

```
```	
/**
	 * \brief 	Sends instantaneously all outgoing bytes
	 * 
	 * \return 	Number of bytes available for writing
	 */	
	virtual void Serial::flush(void) = 0;
```
So it looks the same, no?